vtk/build.sh

```diff
@@ -4,9 +4,13 @@ mkdir build
 cd build
 
 if [ `uname` == Linux ]; then
-    CC=gcc44
-    CXX=g++44
-    PY_LIB="libpython${PY_VER}.so"
+    # Note: VTK 7 requires gcc >= 4.2
+    CC=gcc
+    CXX=g++
```

Comment:
Please leave here gcc44 and g++44. Those are the versions we're using right now in centos5.